saskatoon/harvest/filters.py

```diff
@@ -191,6 +195,35 @@ class Meta:
     )
 
 
+class BeneficiaryFilter(OrganizationFilter):
+
+    def __init__(self, data=None, *args, **kwargs):
+        # set initial value
+        if data is not None:
+            data = data.copy()
+            if 'is_beneficiary' not in data:
+                data['is_beneficiary'] = True
+        super(BeneficiaryFilter, self).__init__(data, *args, **kwargs)
+
+
+class EquipmentPointFilter(OrganizationFilter):
+
+    def __init__(self, data=None, *args, **kwargs):
+        # set initial value
+        if data is not None:
+            data = data.copy()
+            if 'is_equipment_point' not in data:
+                data['is_equipment_point'] = True
+        super(EquipmentPointFilter, self).__init__(data, *args, **kwargs)
+
+    equipment__type = filters.ModelChoiceFilter(
+        queryset=EquipmentType.objects.all(),
+        label=_("Available Equipment Type"),
+        help_text="",
+        required=False
+    )
```

Comment:
mmm I'm not sure I this approach very much, what I would rather we do is:
- filter the respective querysets in `beneficiary_list` and `equipment_point_list` so no unnecessary object is passed to the serializer
- override the Meta class when subclassing OrganizationFilter

Reply:
mentioned this below, but i thought we were set on keeping the querysets identical so we can play with filters on both tabs?